src/XCCDF_POLICY/xccdf_policy_remediate.c

```diff
@@ -802,6 +802,72 @@ static int _xccdf_item_recursive_gather_selected_rules(struct xccdf_policy *poli
 	return ret;
 }
 
+static void _comment_multiline_text_strip_trailing_whitespace(char *token, size_t token_len)
```

Comment:
I think that you can rename this one to `trim_trailing_whitespace` or `rstrip`, as multiline comments are out of scope of this function.

Reply:
OK